dracut.sh

```diff
@@ -661,6 +664,11 @@ while :; do
             PARMS_TO_STORE+=" '$2'"
             shift
             ;;
+        -R | --remove)
```

Comment:
`-R` is accepted here, but not documented anywhere. Since using this option is not recommended, I suggest to remove `-R` and only provide the long option.